jenkins/pipeline/osgp-pr-build.groovy

```diff
@@ -5,7 +5,7 @@ def servername = stream + '-at-pr-' + env.BUILD_NUMBER
 def playbook = stream + '-at.yml'
 
 // Choose the branch to use for SmartSocietyServices/release repository. Default value is 'master'.
-def branchReleaseRepo = 'master'
+def branchReleaseRepo = 'feature/SLIM-1878_Download_meter_keys'
```

Comment:
Note for merge-time. Do not forget to set this back to 'master' at the time the pull request branches are merged.